### ISSUE TITLE
cannot run

### ISSUE THREAD

**Issue by fwt1888** [dwave-examples/job-shop-scheduling#13]
Sorry for disturbing you. I am new to python and I want to run the programme to solve a job-shop problem, but I come across an error. I don't know how to solve it and let the programme run. Could you help me? Thank you very much!
```
Traceback (most recent call last):
  File "C:\Users\14391\Downloads\代码\job-shop-scheduling-master\demo.py", line 31, in <module>
    sampler = EmbeddingComposite(DWaveSampler())
  File "C:\Users\14391\Downloads\代码\job-shop-scheduling-master\venv\lib\site-packages\dwave\system\samplers\dwave_sampler.py", line 161, in __init__
    self.client = Client.from_config(**config)
  File "C:\Users\14391\Downloads\代码\job-shop-scheduling-master\venv\lib\site-packages\dwave\cloud\client.py", line 344, in from_config
    return _clients[_client](**config)
  File "C:\Users\14391\Downloads\代码\job-shop-scheduling-master\venv\lib\site-packages\dwave\cloud\client.py", line 386, in __init__
    raise ValueError("API token not defined")
ValueError: API token not defined
```

**Comment by randomir** [comment]
Hi @fwt1888, looks like you haven't configured Ocean to have access to D-Wave cloud. Namely, you're missing the API token in the config file/environment variable. Please check detailed instructions [in the docs](https://docs.ocean.dwavesys.com/en/stable/overview/sapi.html#sapi-access). If you're new to programming in Python, a good place to start is also our [Getting Started](https://docs.ocean.dwavesys.com/en/stable/getting_started.html) page.

TL;DR: You'll need to sign up for [D-Wave Leap](https://cloud.dwavesys.com/leap/), and then configure Ocean by running `dwave config create` and copy/pasting your API token when asked for it.

Let us know if this helps. Good luck!

**Comment by fwt1888** [comment]
Thanks for your answer ! I will read the docs and try to solve it.
